cpp/pybind/t/geometry/trianglemesh.cpp

```diff
@@ -962,6 +962,18 @@ or has a negative value, it is ignored.
         box = o3d.t.geometry.TriangleMesh.create_box()
         top_face = box.select_by_index([2, 3, 6, 7])
 )");
+    triangle_mesh.def("remove_non_manifold_edges",
```

Comment:
Please add docstrings for python in the [google format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) similar to other functions in this file. Examples are optional but very welcome 😄